frontend/src/app/projects/models/model-source/t4c/service/t4c-model.service.ts

```diff
@@ -79,14 +68,18 @@ export class T4CModelService {
   ): Observable<T4CModel> {
     return this.http.patch<T4CModel>(
       `${this.urlFactory(project_slug, model_slug)}/${t4c_model_id}`,
-      body
+      {
+        t4c_instance_id: body.t4cInstanceId,
+        t4c_repository_id: body.t4cRepositoryId,
+        name: body.name,
+      }
```

Comment:
This could possibly be moved to a function, but since it's only used within this service right now and only twice, it's also fine to leave it that way.

Reply:
Let's keep it for now.